src/mscorlib/corefx/SR.cs

```diff
@@ -815,4 +815,19 @@ internal static string Arg_UnauthorizedAccessException
     {
         get { return Environment.GetResourceString("Arg_UnauthorizedAccessException"); }
     }
+
+    internal static string ArgumentOutOfRange_GenericPositive
+    {
+        get { return Environment.GetResourceString("ArgumentOutOfRange_GenericPositive"); }
```

Comment:
Nit: Not for this PR, but at some point subsequently we might want to go through and replace all of this strings with nameof, e.g.
```C#
using static Environment;
...
internal static string ArgumentOutOfRange_GenericPositive => GetResourceString(nameof(ArgumentOutOfRange_GenericPositive);
```